R/router.R

```diff
@@ -203,9 +205,36 @@ create_router_callback <- function(root, routes = NULL) {
 #'     )
 #'   }
 #' }
+#' \dontrun{
+#'   dynamic_routes <- list(
```

Comment:
Please add an example application that uses dynamic dots to `examples` directory.

Reply:
Added few comments